### PR TITLE
[docs] Add status header for prometheusremotewrite exporter

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
Add status header for prometheusremotewrite exporter.

### CODE REVIEW

File: exporter/prometheusremotewriteexporter/README.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
 # Prometheus Remote Write Exporter
 
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
+| Supported pipeline types | metrics   |
+| Distributions            | [contrib] |
```
Comment (codeboten) [inline exporter/prometheusremotewriteexporter/README.md:7]:
```suggestion
| Status                   |                   |
| ------------------------ |-------------------|
| Stability                | [beta]            |
| Supported pipeline types | metrics           |
| Distributions            | [core], [contrib] |
```

File: exporter/prometheusremotewriteexporter/README.md [hunk 2]
```diff
@@ -73,3 +77,6 @@ Several helper files are leveraged to provide additional capabilities automatica
 - [HTTP settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/confighttp/README.md)
 - [TLS and mTLS settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md)
 - [Retry and timeout settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md), note that the exporter doesn't support `sending_queue` but provides `remote_write_queue`.
+
+[beta]:https://github.com/open-telemetry/opentelemetry-collector#beta
+[contrib]:https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
```
Comment (codeboten) [inline exporter/prometheusremotewriteexporter/README.md:82]:
```suggestion
[contrib]:https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
[core]:https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol
```

**Review by codeboten — COMMENTED** [review]
See suggestions to add core distribution in the header, pinging codeowners to validate beta status @Aneurysm9 @alolita

**Comment by Frapschen** [comment]
done.

**Comment by Frapschen** [comment]
@Aneurysm9 @alolita can you review this PR?

**Review by Aneurysm9 — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Comment by codeboten** [comment]
Re-opening to trigger CI

**Pull request merged**